### PR TITLE
Update UG and bug fixes

### PR DESCRIPTION
**Pull request by timothysashimi**
Closes:
#171
#196 
#234 
#236 
#240 
#241 
#242

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -5,12 +5,8 @@ title: User Guide
 
 FinCliq is a **desktop app for financial advisors to manage contacts and meetings, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, FinCliQ can get your contact management tasks done faster than traditional GUI apps.
 
----
-## Table of contents
-<div class="reset-page-break-defaults" markdown="1">
-* Table of Contents
-{:toc}
-</div>
```
Comment (itstrueitstrueitsrealitsreal) [inline docs/UserGuide.md]: Don't change the toc, I just fixed it 😢

File: docs/UserGuide.md [hunk 2]
```diff
@@ -265,13 +257,6 @@ To keep track of your clients/meetings, you can follow the various commands in t
 **Q**: Are the clients/meetings storage persistent between application restarts?<br>
 **A**: Yes. We have implemented a storage feature that would populate the information inside the address book based on a .json file.
 
-**Q**: When entering my name, some characters do not show up. Is this the intended behaviour?<br>
-**A**: Yes, currently, only our app only supports english names, although we intend to support this in the future.
-
-**Q**: I entered a date and time that was before the current time and it fails to add. Is this
-the intended behaviour?<br>
-**A**: Yes. We implemented this so that people would not enter meetings that occur in the past.
-
```
Comment (itstrueitstrueitsrealitsreal) [inline docs/UserGuide.md]: Can help me add back this part?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -20,8 +16,7 @@ FinCliq is a **desktop app for financial advisors to manage contacts and meeting
 
 1. Download the latest `fincliq.jar` from [here](https://github.com/AY2324S2-CS2103-F08-1/fincliq/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your own personal
-   FinCliq application.
```
Comment (itstrueitstrueitsrealitsreal) [inline docs/UserGuide.md:24]: I think you can add this part back too

File: docs/UserGuide.md [hunk 4]
```diff
@@ -162,26 +157,24 @@ Format: `filter TAG_NAME`
...
+### Adding a Meeting: `addMeeting`
 
 Adds a meeting for a specific client in the address book.
 
 Format: `addMeeting clientIndex/CLIENT_INDEX dt/DATE_TIME d/DESCRIPTION`
 
 - `CLIENT_INDEX` refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​.
-- `DATE_TIME` format should be `YYYY-MM-DD HH:MM`, e.g., `02-01-2025 12:00`. It should also not
-  be before the current date and time.
+- `DATE_TIME` format should be `YYYY-MM-DD HH:MM`, e.g., `02-01-2025 12:00`.
```
Comment (itstrueitstrueitsrealitsreal) [inline docs/UserGuide.md]: This date and time format is incorrect.

**Review by itstrueitstrueitsrealitsreal — COMMENTED** [review]
Just need to make some minor adjustments and tag the relevant issues!

File: docs/UserGuide.md (hunk 1 above)
Comment (timothysashimi) [reply docs/UserGuide.md]: :(

**Pull request merged**